### PR TITLE
Updated PARALLEL description in testJobTemplate

### PR DESCRIPTION
**Pull request by abhijeetjejurkar**
Updated PARALLEL description in testJobTemplate
Fixes: #2943

Signed-off-by: Abhijeet Jejurkar <abhijeet.jejurkar2@gmail.com>

### CODE REVIEW

**Comment by llxia** [comment]
Please remove `This was Updated in AQAvit v0.8.0` in both description and commit message. Thanks

File: buildenv/jenkins/testJobTemplate [hunk 1]
```diff
@@ -383,7 +383,11 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeader('Additional test options if you wish to run in various parallel modes')
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
-							choiceParam('PARALLEL', PARALLEL_LIST, "Optional. Parallel mode")
+							choiceParam('PARALLEL', PARALLEL_LIST, '''Optional. Parallel mode: <br/>
+							None // run test(s) in serial <br/>
+							Dynamic // dynamically divides tests according to NUM_MACHINES and runs them in parallel. Please also provide NUM_MACHINES. <br/>
```
Comment (renfeiw) [inline buildenv/jenkins/testJobTemplate:388]: according to NUM_MACHINES or TEST_TIME

Comment (smlambert) [inline buildenv/jenkins/testJobTemplate:388]:
```suggestion
							Dynamic // dynamically divides tests according to NUM_MACHINES or TEST_TIME and runs them in parallel. Please also provide NUM_MACHINES or TEST_TIME. <br/>
```

**Review by renfeiw — APPROVED** [review]
LGTM

**Comment by smlambert** [comment]
Regen from 2943 branch at: https://ci.adoptopenjdk.net/view/work-in-progress/job/WIP_Test_Job_Auto_Gen/76/ 
Screen Shot 2022-03-22 at 2 21 04 PM

**Pull request merged**